front/src/modules/settings/data-model/objects/SettingsObjectCoverImage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { IconX } from '@/ui/display/icon';
+import { LightIconButton } from '@/ui/input/button/components/LightIconButton';
+import { AnimatedFadeOut } from '@/ui/utilities/animation/components/AnimatedFadeOut';
+import { cookieStorage } from '~/utils/cookie-storage';
+
+import CoverImage from '../assets/build-your-business-logic.jpg';
+
+const StyledCoverImageContainer = styled.div`
+  border: 1px solid ${({ theme }) => theme.border.color.medium};
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  height: 153px;
```

Comment:
As we already have width: 100%, I think we can remove this height, the image should take the right height based on width x its ratio